content/api/commands/session.md

```diff
@@ -0,0 +1,273 @@
+---
+title: session
+---
+
+Use the `session` command to preserve the browser's state between tests. Once the steps to establish a session are performed, the session can be restored at a later time without executing those steps again.
+
+<!--
+Technically it's more the application's state or state *stored* in the browser. The tricky thing here is that we're only preserving cookies and local storage, so saying that we're saving "application state" is misleading. And kind of a bummer because it would be awesome if we did. For now we have app actions for that. Saying that this is a checkpoint or snapshot in time is also similarly ambiguous and misleading.
+-->
+
+A session is persisted by a browser's cookies, [`localStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) and [`sessionStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage).
+
+When your front-end application makes a request and the response contains a `Set-Cookie` header, the browser sets that value as a cookie (or multiple cookies).
+Alternatively, your application may store a value (from a response, user input, etc.) in browser storage (`localStorage` or `sessionStorage`).
+Either way, this persisted data is used by your application and therefore affects its behavior.
```

Comment:
We don't need to mention this here or really at all I this page. If you think it's important to mention this, then we should put it in an info box.

Reply:
I moved this down to Notes until I can figure out how to present it.